src/lib/getConfig.ts

```diff
@@ -0,0 +1,17 @@
+import { SlackChatData, SlackWebhook } from '../types/slack';
+
+export const getConfig = (index: number) => {
+  const url = process.env[`VITE_CHECK_URL_${index}`];
```

Comment:
Looks like the `VITE_` prefix could be irritating. can we make a utility function for this?

Reply:
I think it increases the code complexity: If we suppose that we need a environmental variable without `VITE_ ` prefix. 🤔